locales/es.json

```diff
@@ -199,6 +199,31 @@
     "remove_account": "Eliminar cuenta de la lista",
     "save": "Guardar"
   },
+  "magic_keys": {
+    "dialog_header": "Atajos de teclado",
+    "groups": {
+      "actions": {
+        "boost": "Retootear",
+        "command_mode": "Modo comando",
+        "compose": "Redactar",
+        "favourite": "Favorito",
+        "title": "Acciones",
+        "zen_mode": "Modo Zen"
+      },
+      "media": {
+        "title": "Media"
+      },
+      "navigation": {
+        "go_to_home": "Inicio",
+        "go_to_notifications": "Notificaciones",
+        "next_status": "Siguiente estado",
+        "previous_status": "Anterior estado",
+        "shortcut_help": "Atajo de ayuda",
+        "title": "Navegación"
+      }
+    },
+    "sequence_then": "luego"
```

Comment:
uhmm, esto entiendo que es `key X and then`, debería ser `y después`

Reply:
`y luego` no estaría bien también?

Reply:
deja `seguido de` para ambos, configúralo en `es` y quita la entrada de `es-419`.